src/splitInteger.test.js

```diff
@@ -4,18 +4,30 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const goals = splitInteger(6, 2);
 
+  expect(goals)
```

Comment:
Consider using `.toStrictEqual()` instead of `.toEqual()` for a stricter comparison of array elements. 🧐